src/popup/router/components/Modal.vue

```diff
@@ -44,7 +44,7 @@
         <div
           v-if="$slots.default"
           class="body"
-          :class="{ 'text-center': centered }"
+          :class="{ 'text-center': centered, 'no-padding': bodyNoPadding }"
```

Comment:
No need to add new prop to decrease the body padding as we already have the `noPadding` and `dense` props. What actually you want to achieve? Decrease the space between the buttons and the content? In this case don't use the `#footer` slot for buttons.